data/brands/amenity/restaurant.json

```diff
@@ -45,6 +45,25 @@
     }
   },
   "items": [
+    {
+      "displayName": "Bar-B-Q Plaza",
+      "locationSet": {"include": ["th"]},
+      "matchNames": [
+        "BBQ Plaza",
+        "Bar.B.Q Plaza"
+      ],
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Bar-B-Q Plaza",
```

Comment:
Just double-checking: does the Thailand OSM community expect the primary name to be in English or Thai? From news coverage, it seems like at least some locations primarily signpost the English name, but I couldn’t tell if that’s representative.

Reply:
I was looking at locations in google street view. Most have the Engilsh version largely displayed, but the Thai version is also displayed as a header note or foot note above/below/next to the English. Thus I thought I should add both.

Reply:
Yes, I was just wondering which should be in the main `name` key. Sounds like it should be the English name.